include/clangmetatool/collectors/include_graph.h

```diff
@@ -46,6 +46,20 @@ class IncludeGraph {
    * Get the pointer to the data structure, populated or not.
    */
   IncludeGraphData *getData();
+
+  /**
+   * Get the 'live' dependencies of a header within the given include graph.
+   * This is a subset of the set of transitive dependencies.
+   *
+   * Header B is called a 'live' dependency of B if A uses or references a
+   * name originating from B or any of its transitive dependencies, i.e.:
+   * - A references a macro provided through B
+   * - A references a variable declaration provided through B
+   * - A references a type declaration provided through B
+   * - A redeclares a named declaration in B.
+   */
+  std::set<types::FileUID>
+  liveDependencies(const clangmetatool::types::FileUID &headerFUID) const;
```

Comment:
This should move out of the collector class too, right?

Reply:
Ack.